docs/development/current_measurement.md

```diff
@@ -1,9 +1,177 @@
+
+An amplifier can be connected in either single ended or differential configuration. Single ended configuration is shown in Fig.6a and requires least components and is very simple. But in designs having low value sense resistors or high ground current, it suffers from low accuracy issues. Differential configuration as shown in Fig.6b make use of classic differential amplifier that senses the voltage drop directly across the shunt and hence provides better accuracy than the former.
+
+<figure>
+<center>
+    <img src="./images/low_side_amplifier.svg" height="auto" width="auto" />
+    <figcaption><b>Figure 6.</b> Low-side current measurement using (a) Singled ended and (b) Differential configuration.</figcaption>
+</center>
+</figure>
+
+### 2. High-side current measurement.
+
+In this configuration, the shunt is placed between the supply voltage and the load as shown in Fig.7. This helps to eliminate ground disturbances which was a hurdle in low-side measurement technique and also helps in detecting accidental shorts to system ground. In spite of having this advantage, it suffers from the problem of high common mode voltage. Hence the amplifier used to amplify the small voltage drop across the shunt needs to have specification such that it can tolerate high common-mode voltage (close to supply voltage). This poses a problem in selection of suitable amplifiers. Hence, depending on the application, suitable configuration - either low-side or high-side current measurement technique has to be selected. 
+
+<figure>
+<center>
+    <img src="./images/high_side_current_measurement.svg" height="auto" width="auto" />
+    <figcaption><b>Figure 7.</b> Principle of high-side current measurement.</figcaption>
+</center>
+</figure>
+
+When higher supply voltages (30-40V) are used, the circuit needs to be modified such that low voltage can be measured across the output, so that it can be integrated with other intermediate circuits or microcontroller. The modification depends mainly on the required output volatge range. An example of such circuit is shown below in the Fig.8 where the output voltage has to be within 5V.
+
+<figure>
+<center>
+    <img src="./images/modified_high_side_measurement.svg" height="auto" width="auto" />
+    <figcaption><b>Figure 8.</b> Modified high-side current measurement for high voltages.</figcaption>
+</center>
+</figure>
+
+Note - $V_{dd}$ = $V_{CC}$ + 10
```

Comment:
Use whole sentence: Note that $V_{dd}$ has to be a few volts higher than $V_{cc}$.